sw/airborne/modules/computer_vision/video_thread.h

```diff
@@ -31,19 +31,16 @@
 #define VIDEO_THREAD_H
 
 #include "std.h"
+#include "modules/computer_vision/cv.h"
 
-// Main video_thread structure
-struct video_thread_t {
-  volatile bool is_running;   ///< When the device is running
-  struct v4l2_device *dev;        ///< The V4L2 device that is used for the video stream
-  uint8_t fps;                    ///< The amount of frames per second
-
-  volatile bool take_shot;      ///< Wether to take an image
-  uint16_t shot_number;           ///< The last shot number
-};
-extern struct video_thread_t video_thread;
 
 // Module functions
+struct video_settings camera1_settings;
```

Comment:
Your way to go was a slider to select a camera right? Still think that this would be a bit more strange... what camera gets what number etc. etc.